lib/ssh/src/ssh.erl

```diff
@@ -355,7 +355,7 @@ connect(Host0, Port, UserOptions, NegotiationTimeout)
             {error, Reason};
 
         Options ->
-            SocketOpts = [{active,false} | ?GET_OPT(socket_options,Options)],
```

Comment:
I do not understand, we want the socket option to for sure be {active,false} and the last value added wins (proplist behavior and commandline option behaviour)

>proplists:get_value(active, [{active, false}, {active, true}]).
false

Also probably handle_options is suppose to no include the active option for socket options as it should be emulated.

Reply:
```
1> f(), {ok, ListenSocket} = gen_tcp:listen(8091, [{active,true}, binary, {active, false}]), inet:getopts(ListenSocket, [active]).
{ok,[{active,false}]}
```

Reply:
That was surprising!

Reply:
Or maybe actually the surprising thing is that you could get socket opts here that could already contain  active settings.
Emulated options should not be settable by the user. But as listen works like that you are correct it has to be appended to override the user option if provided.  This however is different form how many of our API functions work as they are perceived as having proplists semantics even if they are handled as proplists internally.

Reply:
But yes you are right ssl:listen behaves the same for the socket options.  Which is not that surprising :) But I have this very strong memory of other options having to behave as proplist.  So sure ssl does Options ++ internal_inet_values() so I guess I changed my mind :)

Reply:
ssl does it in similar way appends active option at the end of the list

```
lib/ssl/src/tls_socket.erl-listen(Transport, Port, #config{transport_info = {Transport, _, _, _, _},
lib/ssl/src/tls_socket.erl-                             inet_user = Options,
lib/ssl/src/tls_socket.erl-                             ssl = SslOpts, emulated = EmOpts} = Config) ->
lib/ssl/src/tls_socket.erl:    case Transport:listen(Port, Options ++ internal_inet_values()) of
lib/ssl/src/tls_socket.erl-     {ok, ListenSocket} ->
```

Reply:
@u3s I think we wrote the last comments somewhat at the same time ...

Reply:
Maybe change the commit message to not say proplist, as it does not behave as a proplist. Coule use option list instead.